app/js/lib/foxbox.js

```diff
@@ -327,6 +326,25 @@ export default class Foxbox extends Service {
     });
   }
 
+  performServiceCommand(id, command, method = 'GET', state = undefined) {
```

Comment:
Mmm, but ```getSnapshot``` is too specific to one of the tons (hopefully) adapters we're going to have. Shouldn't we just have separate ```controllers``` that extend foxbox? Like IP camera controller, Hue controller, Cast controler and etc. Or what is the idea?